src/how-tos/mqtt/mqtt.kafka.broker.md

```diff
@@ -189,11 +230,11 @@ With your `zilla.yaml` config, follow the [Zilla install instructions](../instal
 
 ```yaml:no-line-numbers
 # use the values from $KAFKA_HOST $KAFKA_PORT variables
-env:
+extraEnv:
   - name: KAFKA_HOST
-    value: "\"host.docker.internal\""
+    value: "host.docker.internal"
   - name: KAFKA_PORT
-    value: "\"29092\""
+    value: "29092"
```

Comment:
Do these `extraEnv` var names and values still need to be quoted or can we omit the quotes now, per simple yaml syntax.

Reply:
Yes, the changes we made when adding `extraEnv` removed the need for the extra quotes because the yaml is stitched into the deployment.yaml instead of pulling and assigning values

Reply:
Right, I understand, but meant could we now make it even simpler...
```
value: 29092
```
since the value is implicitly a `string` even without quotes?

Reply:
you would think but that is actually what caused the need for the extra quotes in the first place `"\"\""`. The environment variables map to a go struct that expects a string:
```
Error: Deployment in version "v1" cannot be handled as a Deployment: json: cannot unmarshal number into Go struct field EnvVar.spec.template.spec.containers.env.value of type string
```